lighthouse-core/config/index.js

```diff
@@ -241,19 +305,53 @@ class Config {
       configJSON = defaultConfig;
     }
 
-    this._audits = configJSON.audits ? expandAudits(
-        filterAudits(configJSON.audits, auditWhitelist)
+    this._configJSON = this._initRequirePaths(configJSON);
+
+    this._audits = this.json.audits ? expandAudits(
```

Comment:
Why do we continue if `this.json.audits` is falsy?   I understand for passes & artifacts, but doesn't really make sense here.